manifests/router/link_route_pattern.pp

```diff
@@ -6,10 +6,13 @@
 #
 # $prefix::     Prefix to use
 #
+# $dir::        Direction when using asymmetric routing
```

Comment:
I was on the fence but since you noticed it I'll use direction.

So far I had kept the parameter names from the config file verbatim, but this one is pretty poorly named.